mumax3c/scripts/driver.py

```diff
@@ -32,11 +34,11 @@ def driver_script(driver, system, compute=None, **kwargs):
 
     if isinstance(driver, mc.TimeDriver):
         # Extract dynamics equation parameters.
-        if mm.Precession() in system.dynamics:
+        if system.dynamics.get(type=mm.Precession):
             gamma0 = system.dynamics.precession.gamma0
         else:
             gamma0 = 0
```

Comment:
```suggestion
        gamma0 = precession[0].gamma0 if (precession := system.dynamics.get(type=mm.Precession)) else 0)
```

Reply:
Black will re-format this (line length) in a second commit.

Reply:
I like this C like functionality of assignment in `if` :). Thank you for pointing it out.